script.lua

```diff
@@ -2062,7 +2073,7 @@ function MinorPowerUI(player, button)
     startDealPowerCards({player = player, major = false, count = cards})
 end
 function modifyCardGain(params)
-    for _,obj in pairs(getObjectsWithTag("Modify Card Gain")) do
+    for _,obj in ipairs(getTriggers("Modify Card Gain", "modifyCardGainPriority")) do
```

Comment:
i'm assuming making this `ipairs` is having a different?

Reply:
Yeah, I don't believe `pairs()` places any guarantee on order of iteration, while `ipairs()` does.